templates/webapi/test/overview.html.ep

```diff
@@ -42,7 +42,8 @@
         <div class="card-body">
             % my $allow_commenting = @$job_ids && current_user;
             % if ($allow_commenting) {
-            <button type="button" class="btn btn-secondary btn-circle btn-sm trigger-edit-button" onclick="showAddCommentsDialog()" title="Add comments">
+            <button type="button" class="btn btn-secondary btn-circle btn-sm
+            trigger-edit-button" onclick="showAddCommentsDialog()" title="Add comments or restart Jobs">
```

Comment:
I now realize that there is inconsistent phrasing "Add comments and/or restart [jJ]obs" or "Restart and/or comment jobs". I suggest a consistent phrasing "Restart or comment jobs". Mind the proper casing. I am using "comment" as a verb replacing "Add comments". Also I am using "or" here as inclusive or meaning that it's one of restarting, commenting, restarting && commenting. And please use consistent phrasing in all cases including the button labels and javascript functions.